packages/downloads/package.json

```diff
@@ -10,14 +10,19 @@
   "types": "dist/esm/index.d.ts",
   "license": "Apache-2.0",
   "dependencies": {
-    "@esri/arcgis-rest-feature-layer": "^2.14.1",
-    "@esri/arcgis-rest-portal": "^2.14.1",
-    "@esri/arcgis-rest-request": "^2.14.1",
     "eventemitter3": "^4.0.4",
     "tslib": "^1.13.0"
   },
+  "peerDependencies": {
+    "@esri/arcgis-rest-feature-layer": "^2.21.0",
+    "@esri/arcgis-rest-portal": "^2.21.0",
+    "@esri/arcgis-rest-request": "^2.21.0"
+  },
   "devDependencies": {
-    "@esri/arcgis-rest-auth": "^2.14.1",
+    "@esri/arcgis-rest-auth": "^2.21.0",
```

Comment:
Also, we'll need to update these `devDependencies` in all other packages to match these (even if they're `dependencies`), as per https://github.com/Esri/hub.js/wiki/Dependencies#outside-the-monorepo:

> Then you would bump the version of the dependency in all existing devDependencies within the monorepo to the new minor version.

Reply:
Done.